app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Api
+  module V1
+    class BookSuggestionsController < ApplicationController
+      before_action :authenticate_user!
+      def create
+        new_book_sugges = BookSuggestion.new(book_sugges_params, user: current_user)
+        if new_book_sugges.save
+          render json: new_book_sugges, serializer: BookSuggestionSerializer, status: :created
```

Comment:
Maybe a `return reder if ...` save some code lines

Reply:
Another thing first errors, at the end good response. Read https://github.com/rubocop-hq/ruby-style-guide